src/schema/v2/userInquiredArtworks.ts

```diff
@@ -0,0 +1,47 @@
+import { connectionWithCursorInfo } from "schema/v2/fields/pagination"
+import { ResolverContext } from "types/graphql"
+import {
+  GraphQLString,
+  Thunk,
+  GraphQLFieldConfigMap,
+  GraphQLBoolean,
+} from "graphql"
+import { IDFields } from "./object_identification"
+import { ArtworkType } from "./artwork"
+import { date } from "./fields/date"
+
+export const edgeFields: Thunk<GraphQLFieldConfigMap<
+  any,
+  ResolverContext
+>> = () => ({
+  ...IDFields,
+  status: {
+    type: GraphQLString,
+    description:
+      "This reflects the `title` attribute of the most recent embedded object in `statuses`",
+    resolve: ({ statuses }) => {
+      const statusesWithDates = statuses.map((obj) => {
+        return { ...obj, createdAt: new Date(obj.created_at) }
+      })
+
+      const sortedStatusesReverseChronological = [...statusesWithDates].sort(
+        (objA, objB) => objB.createdAt.getTime() - objA.createdAt.getTime()
+      )
+
+      return sortedStatusesReverseChronological[0]?.title
+    },
+  },
+  outcome: { type: GraphQLString },
+  note: { type: GraphQLString },
+  createdAt: date(({ created_at }) => created_at),
+  isSentToGallery: {
```

Comment:
Just curious if there is any canonical pattern for renaming keys (other than converting from snake to camel casing)?

Reply:
the 'sent to gallery' was sort of my own lingo, since the attribute name `contact_gallery` is a bit vague IMO. It represents whether the inquiry was actually forwarded to the gallery, or whether it went to Artsy (this is all possibly just about legacy data - not sure if this is how it still works, I believe that used to be a gallery-controlled setting).

so it would be `sentToGallery` with the camel-casing, and it's good to prefix bool fields with `is` (if you can remember!), therefore `isSentToGallery`.